docs/pages/build/building-from-github.mdx

```diff
@@ -0,0 +1,116 @@
+  src="/static/images/eas-build/build-from-github/project-github-page.png"
+/>
+
+Install the Expo GitHub App on your GitHub account.
+
+<ImageSpotlight
+  alt="The GitHub app installation UI"
+  src="/static/images/eas-build/build-from-github/install-github-app.png"
+/>
+
+Link your GitHub account to your Expo account.
+
+> **Note:** You must be an admin of the Expo account to install the app. Also, You can only link GitHub
+> organizations to Expo organizations.
+
+<ImageSpotlight
+  alt="The GitHub account linking UI"
+  src="/static/images/eas-build/build-from-github/link-github-account.png"
+/>
+
+Then, you'll be able to link
+your repository to your Expo project.
+
+<ImageSpotlight
+  alt="The repository selector on the Expo project GitHub settings page"
+  src="/static/images/eas-build/build-from-github/connect-a-repository.png"
+/>
+
+### Configure your repository settings
+
+Before running a build, we need to know where to find the source code for your project. If your Expo
+project sourse code is in the root of your repository, then you don't need to do anything. If your
+Expo project source code is in a subdirectory, then you'll need to configure "Base directory"
+settings for your repository on your project's [GitHub settings
+page](https://expo.dev/accounts/[account]/projects/[projectName]/github).
+
+<ImageSpotlight
+  alt="The base directory input on the Expo project GitHub settings page"
+  src="/static/images/eas-build/build-from-github/specify-base-directory.png"
+/>
+
+## Trigger a build from GitHub
+
+Once you've configured your app for GitHub, you can trigger a build from GitHub by using the UI on
+your project's builds list page or by labels on your GitHub PRs.
+
+### Build using the Expo website
+
+Visit your project's [builds list
+page](https://expo.dev/accounts/[account]/projects/[projectName]/builds) and click the "Build from
+GitHub" button. You'll be prompted to select a Git ref (branch/commit/tag), a platform to build
+for and the build profile to apply to it.
+
+You can also specify a base directory for this specific build. That will not change the global
+settings for this project.
+
+<ImageSpotlight
+  alt="The build from GitHub UI on the Expo project builds list page"
+  src="/static/images/eas-build/build-from-github/github-build-ui.png"
+/>
+
+### Build using GitHub PR labels
+
+You can trigger a build from a GitHub PR by adding a label to the PR. The label must be in the form
+of `eas-build-[platform]:[profile]` where `[platform]` is either `android`, `ios`, or `all` and
+`[profile]` is the name of a build profile specified in your **eas.json** file. If you don't specify
+a build platform, it will default to `all`. If you don't speficy
+a build profile, it will default to `production`.
+
+For example, if
+you want to trigger a production build for Android, you would add the label `eas-build-android` to the PR.
+
+<ImageSpotlight
+  alt="A PR with the eas-build label and live status checks"
```

Comment:
```suggestion
  alt="A PR with the EAS Build label and live status checks"
```

Reply:
The intention was to be literal about what the label's contents are. Maybe this suggestion would be clearer:

```suggestion
  alt="A PR with the \"eas-build\" label and live status checks"
```